### ISSUE TITLE
Reconstruction example for keras with tensorflow backend.

### ISSUE THREAD

**Issue by Vin-itall** [kymatio/kymatio#767]
Is there a reconstruction function example for Keras with tensorflow backend?

**Comment by OverLordGoldDragon** [comment]
Only the Torch backend is differentiable - no other backend supports gradient-based reconstruction. Other algorithms exist but there aren't any examples in Kymatio.

**Comment by OverLordGoldDragon** [comment]
Marking as resolved, feel free to open if further concerns.
